arima.py

```diff
@@ -0,0 +1,198 @@
+import numpy as np
```

Comment:
Overall great code quality here, the only comments I have are on documentation which is a great sign. I think commenting what each method is supposed to do would be the main value, especially if this project ends up continuing on and being taken on by new members. This is a very experience team overall, but I think documenting could help if newer members end up joining the team